tests/test_connection_manager.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import mock
+from sceptre.providers.connection_manager import ConnectionManager
+from sceptre.exceptions import ClientError
+from sceptre.exceptions import RetryLimitExceededError
+
+
+class TestConnectionManager(object):
+
+    def test_connection_manager_instantiates_with_config(self):
+        connection_config = {"profile": "prod", "region": "eu-west-1"}
+
+        class ExampleConnectionManager(ConnectionManager):
+            def __init__(self, config):
```

Comment:
You don't need to define the `__init__` method here since it does nothing more than it's parent's init method.

Also a couple more times below

Reply:
👍